### ISSUE TITLE
[BUG] broken undo/redo buttons when there are 2 editors

### ISSUE THREAD

**Issue by RamiroPastor** [HarryChen0506/react-markdown-editor-lite#199]
## OS and browser
Windows 10，Chrome 90.0.4430.93, Firefox 88.0 (64-bit)

## Description
Very wierd behaviour of the undo/redo buttons when there are 2 editors in the same page

## Reproduction URL
https://codesandbox.io/s/upbeat-raman-13vt2?file=/src/App.js

I also made a gyazo of the bug in my original project, here it is:
https://gyazo.com/1e0e3ad6193d6a0cf7c5a94359ea7c23

**Comment by RamiroPastor** [comment]
The button that switches between "only ditor", "only preview" and "both editor and preview" also exhibits a wierd behaviour

**Comment by sylingd** [comment]
Got

**Comment by sylingd** [comment]
I have published 1.2.5-0, please check if the problem has been fixed

**Comment by RamiroPastor** [comment]
Well, i'm not using this package anymore (i was having some troubles integrating with react-hook-form, so i wrote my own markdown editor, which will be of less quality than this one or any other in npm without doubt, especially the undo/redo functionality) but i tried upgrading version to 1.2.5-0 and 1.2.5-1 in the codesandbox i linked in the issue and i get this error:

```
TypeError
Path must be a string. Received null
r
https://codesandbox.io/static/js/common-sandbox.7be24d846.chunk.js:1:246770
Object.isAbsolute
https://codesandbox.io/static/js/common-sandbox.7be24d846.chunk.js:1:247916
(anonymous function)
https://codesandbox.io/static/js/vendors~postcss-compiler~vue-style-compiler.f9e51cefa.chunk.js:1:1069
(anonymous function)
https://codesandbox.io/static/js/vendors~postcss-compiler~vue-style-compiler.f9e51cefa.chunk.js:1:1060
```

**Comment by sylingd** [comment]
It's a bug with CodeSandbox, I fix it at 1.2.5-4

**Comment by RamiroPastor** [comment]
Ok, this problem is fixed. I'll close the issue
